packages/dx-react-grid/src/plugins/table-filter-row.jsx

```diff
@@ -56,17 +56,26 @@ export class TableFilterRow extends React.PureComponent {
                       onValueChange: newValue => onFilter(newValue ? { value: newValue } : null),
                     }}
                   >
-                    {content => (
-                      <FilterCell
-                        {...params}
-                        getMessage={getMessage}
-                        column={params.tableColumn.column}
-                        filter={filter}
-                        onFilter={onFilter}
-                      >
-                        {content}
-                      </FilterCell>
-                    )}
+                    {(content) => {
+                      let cellContent = content;
+                      const filteringEnabled = columnFilteringEnabled(columnName);
+
+                      if (!filteringEnabled) {
+                        cellContent = filter ? filter.value : ' ';
+                      }
+                      return (
+                        <FilterCell
+                          {...params}
+                          getMessage={getMessage}
+                          column={params.tableColumn.column}
+                          filter={filter}
+                          columnFilteringEnabled={filteringEnabled}
```

Comment:
Why do you add a new public interface member? Can we avoid cell rendering when filtering disabled instead?

Reply:
Do you mean render a `TableCell` instead?